src/server/scripts/Northrend/Naxxramas/boss_sapphiron.cpp

```diff
@@ -144,25 +158,31 @@ class boss_sapphiron : public CreatureScript
                 me->CastSpell(me, SPELL_FROST_AURA, true);
 
                 DoCast(me, SPELL_CHECK_RESISTS);
+                events.SetPhase(PHASE_GROUND);
                 events.ScheduleEvent(EVENT_CHECK_RESISTS, 30 * IN_MILLISECONDS);
                 events.ScheduleEvent(EVENT_BERSERK, 15 * MINUTE * IN_MILLISECONDS);
-                EnterPhaseGround();
+                EnterPhaseGround(true);
             }
 
             void SpellHitTarget(Unit* target, SpellInfo const* spell) override
             {
                 switch(spell->Id)
                 {
-                    case SPELL_ICEBOLT:
+                    /*case SPELL_ICEBOLT:
```

Comment:
Yeah, that's old iceblock handling code that I commented for later reference but forgot to remove before pushing this.

I'll get rid of it the next time I work on this.